groalea/graphio.py

```diff
@@ -328,6 +328,21 @@ def Polygon(self, vertices, **kwds):
                     break
         return (pgl.TriangleSet(pgl.Point3Array(p3list), indexlist), None)
 
+	def NURBSCurve(self, ctrlpoints, dimension, **kwds):
+		dimension = int(dimension)
+		points = str(ctrlpoints)
+        points = [float(num) for num in points.split(",")]
+        items, chunk = points, dimension
+        plist = zip(*[iter(items)] * chunk)
+
+        for i in range(len(plist)):
+			ctlplist.append(plist[i]+(1,))
+
+        if dimension == 2:		
+			return (pgl.NURBSCurve2D(ctlplist), None)
+        elif demension == 3:
+			return (pgl.NURBSCurve(ctlplist), None)
+

```

Comment:
@longmanplus 
Problem of indentation:
Please set you editor: indent = 4 spaces

Replace tabs by 4 spaces.